### DISCUSSION TITLE
✨ grimoire

### DISCUSSION THREAD

**Discussion by thescarian** [runtipi/runtipi-appstore#2774] [Requests]
### Description

Bookmark manager with a modern UI, automatic content & metadata extraction, categorization, filtering, and more. It has fully documented REST API, and Docker image for easy deployment.

### Official Website

https://grimoire.pro/

### Documentation link

https://grimoire.pro/docs/intro

### Last application release & date

3.7 (12/8/2023)

### Application license

MIT

### Source code repository link

https://github.com/goniszewski/grimoire

### Docker image link

https://hub.docker.com/r/goniszewski/grimoire

### Please confirm the following

- [X] I confirm that I've put the app name in the title of the discussion
- [X] I confirm that the application is actively maintained.
- [X] I confirm that the docker image of the application is actively maintained.
- [X] I confirm that I have indeed checked that no similar request for this application already exists

### Gathering crowds

Hey everyone! Please take a look at this, it's a super useful tool to store links that you don't wanna bookmark or store in a discord bot DMs lol.

**Comment by sayedhfatimi** [comment]
+1 for grimoire
